Pod/Classes/IPDecimalNumberPadButton.h

```diff
@@ -0,0 +1,13 @@
+//
+//  IPDecimalNumberPadButton.h
+//  Pods
+//
+//  Created by Nicholas Servidio on 10/26/15.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface IPDecimalNumberPadButton : UIButton
```

Comment:
I would actually advocate for leaving a comment here explaining what this button subclass does.